src/utils/bitmap_allocator.rs

```diff
@@ -33,6 +34,32 @@ impl BitmapAllocator64 {
     }
 }
 
+impl BitmapAllocatorTree<BitmapAllocator1024> {
+    pub const fn new() -> Self {
+        Self {
```

Comment:
I think this could work: 
```rust
        Self {
            bits: u16::MAX,
            child: [BitmapAllocator1024::new(); 16],
        }
    }
```